packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -896,8 +971,27 @@ export default function makeKernelKeeper(kernelStorage, kernelSlog) {
       // begin with `vMM.c.o+`.  In addition to deleting the c-list entry, we
       // must also delete the corresponding kernel owner entry for the object,
       // since the object will no longer be accessible.
+      const vref = k.slice(`${vatID}.c.`.length);
+      assert(vref.startsWith('o+'), vref);
```

Comment:
```suggestion
      const vref = k.slice(clistPrefix.length);
      assert(vref.startsWith('o+'), vref);
```
I've also been thinking about introducing a more readable helper for that first step:
```js
      const stripPrefix = (str, prefix) => {
        assert(str.startsWith(prefix));
        return str.slice(prefix.length);
      };
      …
      const vref = stripPrefix(k, clistPrefix);
```

Reply:
yeah, I'll add that, although my brain thinks `stripPrefix(prefix, str)` is the more intuitive order